app/v1/connectors/DeleteCharitableGivingTaxReliefConnector.scala

```diff
@@ -33,10 +33,20 @@ class DeleteCharitableGivingTaxReliefConnector @Inject() (val http: HttpClient,
       hc: HeaderCarrier,
       ec: ExecutionContext,
       correlationId: String): Future[DownstreamOutcome[Unit]] = {
-    post(
-      body = JsObject.empty,
-      DesUri[Unit](s"income-tax/nino/${request.nino}/income-source/charity/annual/${request.taxYear.asDownstream}")
-    )
+
+    import request._
+
+    if (taxYear.useTaxYearSpecificApi) {
+      val downstreamUri = TaxYearSpecificIfsUri[Unit](s"income-tax/${taxYear.asTysDownstream}/$nino/income-source/charity/annual")
+      delete(downstreamUri)
```

Comment:
the 'post' was how it was originally, but I agree it's inconsistent. I've actually taken out the arg names rather than add more, as they're pretty obvious I think - "uri = downstreamUri" seems a bit redundant :)

Reply:
Yeah I agree :)